index.html

```diff
@@ -279,7 +279,7 @@ <h1>About Me</h1>
 </div>
 
 <div class="resumeinsideauto">
-<div class="resume" style="margin-left:122px;">
+<div class="resume" style="margin-left: 122px;">
 <button>Get My Resume</button>
```

Comment:
- [ OPTIONAL ] I think adding a `    border-radius: 4px;` to your button would be better to make them look like the ones in the original design. See the comparison :point_down: 

### Your button 
![Screenshot from 2023-02-24 15-45-47](https://user-images.githubusercontent.com/89228743/221222745-19665030-6634-4978-952e-c04d2cba5d9b.png)

### The original design
![Screenshot from 2023-02-24 15-44-56](https://user-images.githubusercontent.com/89228743/221222722-3e5b9b15-a272-4cf8-aaca-3ef023a721f7.png)

- [ OPTIONAL ] And please for the last `Get in touch` button I would advise removing the border by giving it a `border: none`.

 ### Difference between your button and the original design :point_down: 
### Your button 
![Screenshot from 2023-02-24 15-49-43](https://user-images.githubusercontent.com/89228743/221224417-75b38cf0-8bd3-491a-b757-6c4910186594.png)

### The original design
![Screenshot from 2023-02-24 15-50-16](https://user-images.githubusercontent.com/89228743/221224445-fe5929d5-bb73-47ca-95a1-1f3e9e274022.png)